tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<p>Maybe you can add more explanation here as the pipeline seems long and kinda scary 😅 It can be something like this: "We send all our documents to FileTypeRouter to route them based on their type. Then, we use relevant Converters for each file type to convert them into Haystack Documents. After joining all Documents coming from different Converters with DocumentJoiner, we preprocess them with DocumentCleaner and DocumentSplitter... "  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>That text basically repeats the explanations I had in the code blocks above, though. Do you think it's necessary to explain it twice? if so I can try to find a way to do so that doesn't feel repetitive. 🤔</p>

Reply:
You're right. What you can still emphasize here is the DocumentJoiner. We haven't provided much information about why we need this component, and it makes sense to explain as we create connections. The DocumentJoiner collects documents from different components (in this example, converters) and creates a larger list to pass down the pipeline.